Realm/RLMUtil.hpp

```diff
@@ -17,11 +17,14 @@
 ////////////////////////////////////////////////////////////////////////////
 
 #import <Realm/RLMConstants.h>
-#import <Realm/RLMOptionalBase.h>
+#import <Realm/RLMSwiftValueStorage.h>
+#import <Realm/RLMValue.h>
+
 #import <objc/runtime.h>
 
 #import <realm/array.hpp>
 #import <realm/binary_data.hpp>
+#import <realm/object-store/object.hpp>
```

Comment:
This doesn't appear to do anything which would need this?

Reply:
without this import the compiler complains about not being able to find `realm::CreatePolicy`